webapp/components/notifications/NotificationMenu/NotificationMenu.vue

```diff
@@ -1,23 +1,24 @@
 <template>
-  <ds-button v-if="totalNotifications <= 0" class="notifications-menu" disabled icon="bell">
-    {{ totalNotifications }}
+  <ds-button v-if="!notificationsCount" class="notifications-menu" disabled icon="bell">
+    {{ unreadNotificationsCount }}
   </ds-button>
   <dropdown v-else class="notifications-menu" :placement="placement">
     <template slot="default" slot-scope="{ toggleMenu }">
-      <ds-button primary icon="bell" @click.prevent="toggleMenu">
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-09-25T07:51:23Z" title="Wednesday, September 25th 2019, 9:51:23 am +02:00">Sep 25, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -50,32 +50,20 @@ export default {
   },
   watch: {
     notifications: {
+      immediate: true,
       handler(lastQueriedNotifications) {
         if (lastQueriedNotifications && lastQueriedNotifications.length > 0) {
-          // set this to be empty to get always called if a query comes with results from the backend
-          // has the sideeffect the handler is encouraged to be called again, but only once with no effect, because of the if above
-          this.notifications = []
```


Why you removed this line of code? After our discussion …

@mattwr18 You and I agreed yesterday to leave this line in the code with a specific comment, that I found out that this was needed for a resolution of a misfunctioning I saw, but that I can not exactly say at the moment what this was and we both can not find out right now.

Please don’t do that without talking to me again! 🙏🏼

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-09-25T14:32:33Z" title="Wednesday, September 25th 2019, 4:32:33 pm +02:00">Sep 25, 2019</time>_

---


It wasn't done on purpose and I have no idea how it was deleted... maybe I forgot to save the file before pushing up.
As I didn't do it mindfully, I cannot guarantee that I will not do it again, such it is in life.
Thanks for your understanding

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-09-26T09:09:59Z" title="Thursday, September 26th 2019, 11:09:59 am +02:00">Sep 26, 2019</time>_

---


Okay, I understand and don’t need any guarantee.
If this is not an announcement … 🤣